EmployeeManagementSystem/BusinessLogicLayer/Interface/IDepartmentService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using BusinessLogicLayer.Models;
+
+namespace BusinessLogicLayer.Interface
+{
+    public interface IDepartmentService
+    {
+        List<Department> GetAll();
```

Comment:
please use abstraction instead of implementation (IList, ICollection or IEnumerable)